storage/src/node/path.rs

```diff
@@ -40,6 +40,16 @@ impl Debug for Path {
     }
 }
 
+impl LowerHex for Path {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
+        write!(f, "0x")?;
+        for nib in &self.0 {
+            write!(f, "{:x}", *nib)?;
+        }
```

Comment:
I would probably delegate. Alignment, fill, and precision are not handled, and probably should be.

I'm okay with leaving a TODO on whatever it is you aren't implementing yet.
```suggestion
    let mut iter = self.0.iter();
        let Some(first) = iter.next() else {
            return Ok(())
        };
        fmt::LowerHex::fmt(first, f)?;
        while let Some(next) = iter.next() {
            write!(f, "{:x}", next)?;
        }
        Ok(())
```

Reply:
I don't think we will ever want to print nibbles with precisions or `0x` prefix but will leave a todo here if there is a real need.